.github/workflows/tag.yml

```diff
@@ -90,3 +90,41 @@ jobs:
             golangci/golangci-lint:${{ steps.prepare.outputs.full_tag_name }}
             golangci/golangci-lint:${{ steps.prepare.outputs.full_major_tag }}
             golangci/golangci-lint:${{ steps.prepare.outputs.latest_tag }}
+
+  update-docs-and-assets:
+    needs: docker-release
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GOLANGCI_LINT_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Update GitHub action config
+        run: make assets/github-action-config.json
```

Comment:
This step requires that the GitHub release was done, so I think that we have to handle all the post-release on another event that a tag

Reply:
actually it's non needed since the `need` parameter was used. But it's OK